pkg/report/linux.go

```diff
@@ -426,7 +437,7 @@ func (ctx *linux) symbolize(rep *Report) error {
 }
 
 func symbolizeLine(symbFunc func(bin string, pc uint64) ([]symbolizer.Frame, error),
-	symbols map[string][]symbolizer.Symbol, vmlinux, strip string, line []byte) []byte {
+	symbols map[string]map[string][]symbolizer.Symbol, modules []*vminfo.KernelModule, ctx *linux, line []byte) []byte {
```

Comment:
This patch does not compile, because you remove `strip` here, but it is still used below. Please fix.

Reply:
Overall, please make sure that syzkaller builds after each individual commit in the series.